From ee11ea660a863c0844ec1ce67a9f36ce30cda8a6 Mon Sep 17 00:00:00 2001
From: Zoltan Gilian <zoltan.gilian@gmail.com>
Date: Wed, 16 Aug 2017 22:54:31 +0200
Subject: [PATCH] Fix undefined TBB_LIBRARYDIR on Windows in FindTBB.cmake.

---
 cmake/FindTBB.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/FindTBB.cmake b/cmake/FindTBB.cmake
index 3fe6446..590c7e8 100644
--- a/cmake/FindTBB.cmake
+++ b/cmake/FindTBB.cmake
@@ -79,6 +79,7 @@ IF ( TBB_FOUND )
       SET ( TBB_VC_DIR vc12 )
 	ENDIF ( MSVC10)
 	#  MESSAGE ( "TBB_VC_DIR = ${TBB_VC_DIR}" )
+	SET ( TBB_LIBRARYDIR ${TBB_LOCATION}/lib CACHE STRING "TBB library directory")
 	SET (TBB_PATH_SUFFIXES intel64/${TBB_VC_DIR} )
 	FIND_LIBRARY ( TBB_LIBRARY_PATH tbb PATHS ${TBB_LIBRARYDIR} PATH_SUFFIXES ${TBB_PATH_SUFFIXES})
 	FIND_LIBRARY ( TBB_PREVIEW_LIBRARY_PATH tbb_preview PATHS ${TBB_LIBRARYDIR}  PATH_SUFFIXES ${TBB_PATH_SUFFIXES})
-- 
2.8.1.windows.1

